modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/v2/form_data.rb

```diff
@@ -35,7 +35,11 @@ def mailing_address
         end
 
         def preferred_email
-          signing_appellant.email.presence
+          long_preferred_email? ? 'See attached page for preferred email' : signing_appellant.email.presence
```

Comment:
I see what you're saying. I used the same text printed on v1. The fallback text ended up at the front after I refactored to check for a 'long' preferred instead of 'short' preferred for other reasons. I agree the fallback at the start is distracting and annoying, but which is more annoying...that, or `!longpreferred_email?`. I don't know. But, falling back to a guard clause? Not having any directive text in the box? I'm not sure I understand.

Reply:
Refactored a bit. Let me know what you think!